functions/__fzf_search_current_dir.fish

```diff
@@ -17,7 +17,7 @@ function __fzf_search_current_dir --description "Search the current directory us
 
         for path in $file_paths_selected
             set escaped_path (string escape "$path")
-            commandline --insert "$escaped_path "
+            commandline --current-token --replace "$escaped_path "
```

Comment:
Sorry to be nitpicky--I know this logic works--but it's a bit hacky. The only reason `commandline --current-token --replace` doesn't keep overwriting itself and instead inserts is because we padded `escaped_path` with a space. I'm currently brainstorming how we can make this code be less deceptive / easier to follow...
One idea off the top of my head is to map `file_paths_selected` using string escape and then use string join on it. Then we can do `commandline --current-token --replace $joined_escaped_paths`

Reply:
Wow, I didn't notice that.